share/etc/oned.conf

```diff
@@ -185,13 +185,13 @@ RAFT = [
 # Executed when a server transits from follower->leader
 # RAFT_LEADER_HOOK = [
 #     COMMAND = "raft/vip.sh",
-#     ARGUMENTS = "leader <interface> <ip_cidr>"
+#     ARGUMENTS = "leader interface:ip_cidr [interface:ip_cidr ...]"
```

Comment:
This is unfortunate for upgrades (also might not handle properly IPv6 addresses with `:` inside?)

Can we better consider just arguments extension without introducing breaking change? E.g.,
`leader interface ip_cidr interface2 ip_cidr2 interface3 ip_cidr3`